### DISCUSSION TITLE
Grammatical error/ spelling mistake in the website

### DISCUSSION THREAD

**Discussion by Shalini469717** [pretix/pretix#3169] [General]
![image](https://user-images.githubusercontent.com/99305545/226485670-64cbbdb8-8b6a-4337-89cb-767701903cf8.png)
 > From an **organizer** point of view there is nothing left to be desired.
 
Shouldn't it be:
From an *organizer's* point of view *,* there is nothing left to be desired.

**Comment by raphaelm** [comment]
Thank you, will fix soon!
